config/crs/openshift/pruner/operator_v1alpha1_pruner_cr.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2025 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonPruner
+metadata:
+  name: pruner
+spec:
+  targetNamespace: openshift-pipelines
+  pruner:
+    disabled: true
```

Comment:
Hey just went through other configurations like Chains, Results and i see its managed in code itself for the default values I mean we no need to explicitly set in `operator_v1alpha1_pruner_cr.yaml` file

Can you check how other components working and change accordingly